s30310-KacperKoszelak/zajecia3/3_3.txt

```diff
@@ -5,7 +5,7 @@ terminalu. Proszę go przetestować w taki sposób, że:
 
 1. Uruchom komendę cat be parametrów
 2. Wciśnij CTRL+C i zobacz co się stanie
-    -zakończyło się wykonywanie komendy cat
+    -zakończyło się wykonywanie komendy cat, program wypisał komunikat "^C"

```

Comment:
To nie program tylko `bash` w konsoli wstawił tekstową reprezentację tejże kombinacji.